src/main/java/com/mixpanel/android/mpmetrics/AnalyticsMessages.java

```diff
@@ -233,7 +235,11 @@ public void handleMessage(Message msg) {
                         logAboutMessageToMixpanel("Queuing people record for sending later");
                         logAboutMessageToMixpanel("    " + message.toString());
 
-                        queueDepth = mDbAdapter.addJSON(message, MPDbAdapter.Table.PEOPLE);
+                        if (mDbAdapter.belowMemThreshold()) {
+                            queueDepth = mDbAdapter.addJSON(message, MPDbAdapter.Table.PEOPLE);
+                        } else {
+                            Log.e(LOGTAG, "Dropping events queries due to short of memory");
```

Comment:
Since these error messages will be visible to the user, we might want to be super specific- perhaps something like
```
"There is not enough space left on the device to store Mixpanel data, so a people analytics update was not sent"
```